src/adhocracy_core/adhocracy_core/workflows/debate_private.yaml

```diff
@@ -9,7 +9,7 @@ states:
     acm:
       principals:       [everyone,  participant, moderator, creator, initiator, admin]
       permissions:
-        - [view,         D,         ~,           ~,          ~,      ~,         ~]
+        - [view,         D,         A,           A,         A,       A,         ~]
```

Comment:
This should result in admins not still not beeing able to see those processes? Is that intended?